kgl-math/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+	kotlin("multiplatform")
+	`maven-publish`
+}
+
+kotlin {
+	sourceSets {
+		commonMain {
+			dependencies {
+				implementation(kotlin("stdlib-common"))
+			}
+		}
+		commonTest {
+			dependencies {
+				implementation(kotlin("test-common"))
+				implementation(kotlin("test-annotation"))
+			}
+		}
+	}
```

Comment:
Will have to list out all possible targets here. At least until [KT-30088](https://youtrack.jetbrains.com/issue/KT-30088) is resolved.

Reply:
Is it possible to easily define *all* presets? Since this will be a Common library, all targets are possible.

Reply:
Something like this is possible.
```
kotlin {
    presets.forEach {
        targetFromPreset(it) { 
            /* Configure each of the created targets */
        }
    }
}
```
Gotten from [`here`](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#setting-up-targets).

Reply:
wasm target is possible, as long as exceptions aren't thrown (too often), because [this](https://github.com/JetBrains/kotlin-native/issues/3194). That detail could be ignored for now.

Reply:
implemented in newest commit effc8c5b86bb6c72ee14721053746fcd3a4b1790. I configured the JVM and JS targets separately since they need an explicit stdlib dependency.